examples/starrocks/deploy_a_starrocks_cluster_with_share_data_mode.yaml

```diff
@@ -0,0 +1,60 @@
+# This manifest deploys a StarRocks cluster running in share data mode.
```

Comment:
provide a link to our docs (https://docs.starrocks.io/en-us/latest/deployment/deploy_shared_data), so the user will know better about what is a `shared_data` mode.